Orange/widgets/unsupervised/tests/test_owtsne.py

```diff
@@ -87,7 +87,15 @@ def test_wrong_input(self):
         self.send_signal(self.widget.Inputs.data, self.data)
         self.wait_until_stop_blocking()
         self.assertIsNone(self.widget.data)
-        self.assertTrue(self.widget.Error.no_attributes.is_shown())
+        self.assertTrue(self.widget.Error.not_enough_cols.is_shown())
+
+        # one attributes
+        self.data = Table.from_list(self.empty_domain, [[1, 'STG1'],
```

Comment:
@rokgomiscek, pylint complains about `self.data` being defined outside constructor. This doesn't happen just in your code but throughout this function. Could you change this to a local variable in all places? I assume this doesn't need to be an attribute.

Reply:
@janezd, changed